{"name": "Create order without auth", "status": "broken", "statusDetails": {"message": "NameError: name 'Urls' is not defined", "trace": "self = <test_create_order.TestCreateOrder object at 0x000001EC6313BB10>, api_client = <helpers.api_client.APIClient object at 0x000001EC62EF6E90>\n\n    @allure.title('Create order without auth')\n    def test_create_order_no_auth(self, api_client):\n>       ingredients = get_valid_ingredients(api_client)\n\ntests\\test_create_order.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\napi_client = <helpers.api_client.APIClient object at 0x000001EC62EF6E90>\n\n    def get_valid_ingredients(api_client):\n>       response = api_client.get(Urls.INGREDIENTS)\nE       NameError: name 'Urls' is not defined\n\nhelpers\\data_generator.py:16: NameError"}, "start": 1755429580834, "stop": 1755429580834, "uuid": "6fecd57b-0402-4947-a3ff-b76c2d1404b3", "historyId": "6ac0499286a8043701f4341522cff5ed", "testCaseId": "6ac0499286a8043701f4341522cff5ed", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_no_auth", "labels": [{"name": "feature", "value": "Order Creation"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "dea-1611"}, {"name": "thread", "value": "12860-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}